kv/union_store.go

```diff
@@ -127,7 +127,7 @@ func (us *UnionStore) CheckLazyConditionPairs() error {
 	for it := us.lazyConditionPairs.NewIterator(nil); it.Valid(); it.Next() {
 		if len(it.Value()) == 0 {
 			if _, exist := values[it.Key()]; exist {
-				return errors.Trace(ErrKeyExists)
```

Comment:
I think now the best way is to use juju errors.Annotate(kv.ErrKeyExist, message) maybe.